nixos/modules/services/cluster/kubernetes/dns.nix

```diff
@@ -0,0 +1,240 @@
+{ cfg }: {
```

Comment:
If they're fully generated and this is in a module, it might make sense to just import them as straight JSON, possibly even referring to the external file? I don't think import-from-derivation is that big of a deal in system configurations.